main.py

```diff
@@ -1,10 +1,71 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    races = {}
+    guilds = {}
+    skills_to_create = []
+    players_to_create = []
+
+    for player in data:
+        race_name = data[player]["race"]["name"]
+        race_description = data[player]["race"]["description"]
+        if race_name not in races:
+            races[race_name] = Race(
+                name=race_name, description=race_description)
+
+        guild_data = data[player].get("guild")
+        if guild_data:
+            guild_name = guild_data["name"]
+            guild_description = guild_data["description"]
+            if guild_name not in guilds:
+                guilds[guild_name] = Guild(
+                    name=guild_name, description=guild_description)
+
+    # Save races and guilds first
+    Race.objects.bulk_create(races.values(), ignore_conflicts=True)
```

Comment:
This is a performance issue. You are using `bulk_create` method with `ignore_conflicts=True` option. This means that if there is a conflict, Django will ignore it and continue. This can lead to data inconsistency. You should handle conflicts properly. For example, you can use `update_or_create` method to update existing objects or create new ones.